### DISCUSSION TITLE
What is the process for using a custom pretrained model?

### DISCUSSION THREAD

**Discussion by Jcwscience** [coqui-ai/TTS#1684] [General Q&A]
I have a tacotron2 model that I trained on my own voice, and I’d like to use the inference engine with it. How would I go about doing that? I know there is a way to select your own model in the command line options, but it seems to need more config information?

**Comment by p0p4k** [comment]
Hello, you just replace `--model_name` with `--model_path <path to your trainded model.pth>` and `--config_path <path to config.json>`.
And as always, you can refer to the [official docs](https://tts.readthedocs.io/en/latest/inference.html) before posting questions. Thanks.
